mkdocs.yml

```diff
@@ -4,7 +4,7 @@ theme: readthedocs
 repo_url: https://github.com/FakeItEasy/FakeItEasy
 extra_css: [css/extra-highlight.css]
 
-pages:
+nav:

```

Comment:
I got a warning when I ran locally, so I updated. I could pretend that this is the change that lets me link to a URL instead of a relative .md file down on line 40, but honestly, I did not try.